extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/thing/thing-types.xml

```diff
@@ -26,5 +27,12 @@
         <description>Selects the color of the light</description>
         <category>ColorLight</category>
     </channel-type>
+    
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
```

Comment:
Thanks for the offer @hadees, but I think @kgoderis is on it and it should be easy for him to fix with the next update of this PR.